Formula/sslscan.rb

```diff
@@ -17,10 +17,12 @@ class Sslscan < Formula
 
   def install
     system "make"
+    mkdir_p [bin, man1]
```

Comment:
This seems like a regression; can you open a ticket against the sslscan repository and link to it here? Thanks!